velox/substrait/SubstraitParser.h

```diff
@@ -49,6 +49,9 @@ class SubstraitParser {
   std::shared_ptr<SubstraitType> parseType(
       const ::substrait::Type& substraitType);
 
+  // input type like i32
```

Comment:
It would be better to follow Velox comment style. That is, comments should be full english sentences, starting with a capital letter and ending with a period (.).
Link: https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md.